panther_analysis_tool/main.py

```diff
@@ -1239,6 +1264,10 @@ def _run_tests(  # pylint: disable=too-many-arguments
             logging.debug(str(err), exc_info=err)
             failed_tests[detection.detection_id].append(unit_test["Name"])
             continue
+        finally:
```

Comment:
Noting for reviewers that `finally` will run even when an `except` block contains a `continue`: https://docs.python.org/3/reference/compound_stmts.html#finally